azure/pipelines/dotnet/ci.yml

```diff
@@ -33,10 +33,14 @@ parameters:
   type: boolean
   default: false
   displayName: 'Whether to run ''dotnet package''.'
+- name: WithPush
+  type: boolean
+  default: false
+  displayName: 'Whether to run push NuGet packages to external feed.'
 - name: WithPublish
   type: boolean
   default: false
-  displayName: 'Whether to run ''dotnet publish''.'
+  displayName: 'Whether to run ''dotnet publish'' and publish to pipeline artifacts.'
```

Comment:
What if you only want to "publish" because you want to spin up an IIS Express with you webapp for testing during CI? Example: ODS postman tests. You want to run those tests during PR build, but you don't want to push artifacts during PR build.